test/create_lira_config.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import re
+
+def run(args, env_config, secrets_config):
+    pipeline_tools_prefix = args.pipeline_tools_prefix
+    tenx_prefix = args.tenx_prefix
+    ss2_prefix = args.ss2_prefix
+
+    config = {
+        'env': env_config['env'],
+        'cromwell_url': 'https://cromwell.mint-{0}.broadinstitute.org/api/workflows/v1'.format(env_config['env']),
+        'MAX_CONTENT_LENGTH': 10000,
+        'submit_wdl': '{0}/adapter_pipelines/submit.wdl'.format(pipeline_tools_prefix),
+    }
+    config.update(secrets_config)
+
+    wdl_config = {
+      'wdls': [
+        {
+          'subscription_id': env_config['10x_subscription_id'],
+          'workflow_name': 'Adapter10xCount',
+          'analysis_wdl': '{0}/10x/count/count.wdl'.format(tenx_prefix),
+          'wdl_link': '{0}/adapter_pipelines/10x/adapter.wdl'.format(pipeline_tools_prefix),
+          'wdl_default_inputs_link': '{0}/adapter_pipelines/10x/adapter_example_static.json'.format(pipeline_tools_prefix),
+          'options_link': '{0}/adapter_pipelines/10x/options.json'.format(pipeline_tools_prefix)
+        },
+        {
+          'subscription_id': env_config['ss2_subscription_id'],
+          'workflow_name': 'AdapterSs2RsemSingleSample',
+          'analysis_wdl': '{0}/smartseq2_single_sample/ss2_single_sample.wdl'.format(ss2_prefix),
+          'wdl_link': '{0}/adapter_pipelines/smart_seq2/adapter.wdl'.format(pipeline_tools_prefix),
+          'wdl_default_inputs_link': '{0}/adapter_pipelines/smart_seq2/adapter_example_static_demo.json'.format(pipeline_tools_prefix),
+          'options_link': '{0}/adapter_pipelines/smart_seq2/options.json'.format(pipeline_tools_prefix)
+        }
+      ]
+    }
+    config.update(wdl_config)
+    print(json.dumps(config, indent=2, sort_keys=True))
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--env_config_file', required=True)
```

Comment:
If we are going to only use `dev` and `int` for this local integration test, it might be great to check the inputs here, instead of throwing runtime error (such as KeyError) later.

Reply:
Also, do you think we could tag the current master branch of lira as `0.2.0` if you agree with the version number?

Reply:
I tagged version 0.2.0 of Lira as you suggested. I'm not sure I follow what you said about checking the inputs here. We are just letting the user pass in the path to the environment config file they want to use (which just has the name of the environment plus the subscription ids). I don't see a need to restrict what values are legal inside that file. This will leave it more flexible in case anyone ever wants to run this in staging for example.

Reply:
If that is the case, this makes much sense to me. Thanks for the explanation!!